src/main/java/com/strandls/document/controllers/DocumentController.java

```diff
@@ -96,6 +100,58 @@ public Response getPong() {
 		return Response.status(Status.OK).entity("PONG").build();
 	}
 
+	@PUT
+	@Path(ApiConstants.DELETENAME + "/{nameId}")
+	@Consumes(MediaType.TEXT_PLAIN)
+	@Produces(MediaType.APPLICATION_JSON)
+
+	@ApiOperation(value = "delete scientific name detected for a document by gnfinder", notes = "returns the object with updated column", response = DocSciName[].class)
```

Comment:
is the response a docSciname array or a single object. Please review the response type

Reply:
Corrected.